roles/edpm_update_system/tasks/bootc.yml

```diff
@@ -18,8 +18,31 @@
 - name: Update bootc image
   ansible.builtin.shell: bootc switch "{{ edpm_update_system_bootc_os_container_image }}"
   become: true
-  register: bootc_result
-  changed_when: bootc_result.rc != 0
+  register: bootc_switch_result
+  failed_when: bootc_switch_result.rc != 0
+  changed_when: false
```

Comment:
I think we can drop this changed_when.
Unless we want to compare the staged value before and after the switch command, and if the value has changed, then set changed_when based on that.

Reply:
We can also drop the `bootc_switch_result` altogether. If the command returns RC != 0, it will fail the task anyway, we don't need to register that var. It's not used anywhere else. (Did that in https://github.com/openstack-k8s-operators/edpm-ansible/pull/1069/files )

Reply:
Yeah, it makes sense. I checked the PR and also realized I'm never using it anywhere and is not required.